neuro_flow/batch_runner.py

```diff
@@ -451,6 +482,12 @@ async def _process_started(
                 click.echo(f"Action {str_full_id} is {str_status}")
                 parent_ctx, parent_topo = topos[st.id[:-1]]
                 parent_topo.done(st.id)
+                if (
+                    finished[st.id].status != JobStatus.SUCCEEDED
```

Comment:
Seems like I found an issue here. Action can get a DISABLED state if it has some disabled task. Currently this will cause assertion error in the `.finish_batch_action`. 
It's my bad, so no action from you is required in this PR.

Reply:
ok